packages/asset-uploader/jest.config.js

```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line
-const base = require("jest-config/jest.config.js");
+const base = require("@webstudio-is/jest-config/jest.config.js");
```

Comment:
shouldn't this be, couldrename jest.config.js to index.js
```suggestion
const base = require("@webstudio-is/jest-config");
```

Reply:
Fixed it.